server_installation/topics/operating-mode/crossdc/proc-configuring-remote-cache.adoc

```diff
@@ -111,21 +113,133 @@ For example, add a cache named `sessions` with the following configuration:
+. Start `NODE22` :
++
+[source,subs="+quotes"]
+----
+cd NODE22/bin
+./standalone.sh -c standalone-ha.xml -Djboss.node.name=node22 -Djboss.site.name=site2 \
+  -Djboss.default.multicast.address=234.56.78.2 -Dremote.cache.host=server2 \
+  -Djava.net.preferIPv4Stack=true -b _PUBLIC_IP_ADDRESS_
+----
++
+It should be in cluster with `NODE21` :
++
+```
+Received new cluster view for channel keycloak: [node21|1] (2) [node21, node22]
+```
++
+
+NOTE: The channel name in the log might be different.
+
+. Test:
+
+.. Go to `http://node11:8080/auth/` and create the initial admin user.
+
+.. Go to `http://node11:8080/auth/admin` and login as admin to admin console.
+
+.. Open a second browser and go to any of nodes `http://node12:8080/auth/admin` or `http://node21:8080/auth/admin` or `http://node22:8080/auth/admin`. After login, you should be able to see
+the same sessions in tab `Sessions` of particular user, client or realm on all 4 servers.
+
+.. After doing any change in Keycloak admin console (eg. update some user or some realm), the update
+should be immediately visible on any of 4 nodes as caches should be properly invalidated everywhere.
+
+.. Check server.logs if needed. After login or logout, the message like this should be on all the nodes `NODEXY/standalone/log/server.log` :
++
+```
+2017-08-25 17:35:17,737 DEBUG [org.keycloak.models.sessions.infinispan.remotestore.RemoteCacheSessionListener] (Client-Listener-sessions-30012a77422542f5) Received event from remote store.
+Event 'CLIENT_CACHE_ENTRY_REMOVED', key '193489e7-e2bc-4069-afe8-f1dfa73084ea', skip 'false'
+```
+
+
+. Start each {project_name} server, for example:
++
+.Site1
+[source,xml,options="nowrap",subs=attributes+]
+----
+./standalone.sh -c standalone-ha.xml \
+                -Djboss.node.name=node11 \
+                -Djboss.site.name=site1 \
+                -Djboss.default.multicast.address=234.56.78.1 \
+                -Dremote.cache.host=server1 \
+                -Djava.net.preferIPv4Stack=true \
+                -b _PUBLIC_IP_ADDRESS_
+----
++
+.Site2
+[source,xml,options="nowrap",subs=attributes+]
+----
+./standalone.sh -c standalone-ha.xml \
+                -Djboss.node.name=node12 \
+                -Djboss.site.name=site2 \
+                -Djboss.default.multicast.address=234.56.78.2 \
+                -Dremote.cache.host=server1 \
+                -Djava.net.preferIPv4Stack=true \
+                -b _PUBLIC_IP_ADDRESS_
+----
++
+If you notice the following warning messages in logs, you can safely ignore them:
++
+[source,options="nowrap",subs=attributes+]
+----
+WARN  [org.infinispan.CONFIG] (MSC service thread 1-5) ISPN000292: Unrecognized attribute 'infinispan.client.hotrod.auth_password'. Please check your configuration. Ignoring!
+WARN  [org.infinispan.CONFIG] (MSC service thread 1-5) ISPN000292: Unrecognized attribute 'infinispan.client.hotrod.auth_username'. Please check your configuration. Ignoring!
 ----
 +
 . Check server logs for the following messages:
```

Comment:
The step 13. "Check server logs for the following messages: ... " should be **removed** now.